packages/label-sync/src/label-sync.ts

```diff
@@ -281,6 +281,9 @@ async function reconcileLabels(
     'kokoro:force-ci',
     'kokoro: force-run',
     'kokoro: run',
+    'buildcop: issue',
+    'buildcop: quiet',
+    'buildcop: flaky',
```

Comment:
We should also replace these with the corresponding new label.

Reply:
That's already done!  We added the new `flakybot: ` labels in #1306, and I manually just went through and replaced all instances of `buildcop: ` labels with the new auto created `flakybot: ` labels.

Reply:
Ah, forgot we were doing that manually. Thanks!